Samples.Objc/Samples.ObjC/ViewControllers/CheckoutViewController/Views/ItemCell.m

```diff
@@ -0,0 +1,82 @@
+//
+//  ItemCell.m
+//  PayPalNativeCheckoutObjC
+//
+//  Created by Nguyen, The Nhat Minh on 3/9/21.
+//  Copyright © 2020 PayPal. All rights reserved.
+//
+
+#import "ItemCell.h"
+
+@interface ItemCell ()
+
+@property (nonatomic) UIStackView *stackView;
+@property (nonatomic) UILabel *nameLabel;
+@property (nonatomic) UILabel *priceLabel;
+@property (nonatomic) UILabel *taxLabel;
+
+@end
+
+@implementation ItemCell
+
+- (instancetype)initWithItem:(PPCPurchaseUnitItem *)item andQuantity:(NSString *)quantity {
+  self = [super init];
+  self.selectionStyle = UITableViewCellAccessoryNone;
+  [self setupUIWithItem:item andQuantity:quantity];
+  [self setupConstraints];
+  return self;
+}
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+}
+
+- (void)setupUIWithItem:(PPCPurchaseUnitItem *)item andQuantity:(NSString *)quantity {
+  self.nameLabel = [[UILabel alloc] init];
+  self.nameLabel.textColor = [UIColor blackColor];
+  self.nameLabel.font = [UIFont systemFontOfSize:16];
+  self.nameLabel.text = [NSString stringWithFormat:@"%@ x %@", quantity, item.name];
+  [self.nameLabel sizeToFit];
```

Comment:
We repeat this same set of formatting lines several times - maybe worth a convenience function to generate / format your label? 
ie
```objc
 (UILabel *)createAndFormatLabelWithText:(NSString *)text fontSize:(NSInteger)fontSize {
  UILabel * label = [UILabel new];
  label.textColor = [UIColor blackColor];
  label.font = [UIFont systemFontOfSize: fontSize];
  label.text = text;
  return label;
}
```